src/main/java/com/bulkbuy/enterprise/BulkBuyController.java

```diff
@@ -15,14 +22,22 @@ public String index() {
         return "start";
     }
 
-    @RequestMapping("/newOrder")
-    public String newOrder(){
+    @GetMapping("/newOrder")
+    public String newOrder(Model model){
+        model.addAttribute("order", new Order());
         return "newOrder";
     }
 
+    @PostMapping(value="/saveOrder")
+    public String saveOrder(@ModelAttribute("order") Order order, Model model) {
+        Order newOrder = null;
+        return "start";
+    }
+
```

Comment:
Added GetMapping and PostMapping for the new order form page. This was needed to be able to link to the html page using form submit buttons and to render the page without error.  This should help with having smart endpoints.
https://martinfowler.com/articles/microservices.html